tests/compute/vmi-configuration/vmi-definition.go

```diff
@@ -1019,7 +1019,7 @@ var _ = ConfigDescribe("VirtualMachineInstance definition", func() {
 		var rngVmi *v1.VirtualMachineInstance
 
 		BeforeEach(func() {
-			rngVmi = libvmi.NewAlpine(withNoRng())
+			rngVmi = libvmi.NewAlpine(libvmi.WithNoRNG())
```

Comment:
What is this oxymoron? rngVmi is a alpine vmi without rng! :confused: 
I know it was there before this PR, but can we please fix it :pray: ? Maybe using `vmi` instead of new `rngVmi`

Reply:
Done